site/content/docs/contributing/project-structure.md

```diff
@@ -9,42 +9,139 @@ menu:
 
 🚧 This is a work-in-progress 🚧
 
-## CLI
-```
-.
-├── cmd
-│   └── kind
-│       ├── build/   # Build images
-│       ├── create/  # Create cluster
-│       ├── delete/  # Delete cluster
-│       ├── get/     # List kubeconfigs and clusters
-│       └── kind.go  # Root command
-├── main.go          # Entrypoint
-```
+## Code Layout
+The kind project is composed of two parts: the CLI and the packages that        
+implement kind's functionality.
+We will go more in depth below.
+
+### CLI
 
-The CLI is built using [cobra][cobra] and you can see the app's entrypoint, [`main.go`][main.go], at the root level of the repository.
-The CLI commands can be found in the directory [cmd][cmd]. Here, you will find
-the root command [kind.go][kind.go] where we register other commands to build
-images; create, delete, and list clusters; list kubeconfig files for cluster;
-and setup logging.
+kind's CLI commands are defined in [cmd/kind](/cmd/kind).
+Each subdirectory corresponds to a command, and each of their subdirectories
+implements a subcommand.
+The CLI is built using [cobra][cobra] and you can see the app's entrypoint,
```

Comment:
looks like this is meant to say `in kind.go`?

Reply:
Fixed this link